package.json

```diff
@@ -34,18 +35,21 @@
   "devDependencies": {
     "async": "^2.2.0",
     "bit-docs": "0.0.7",
-    "can-vdom": "^3.1.0-pre.1",
+    "can-simple-dom": "^1.1.1",
```

Comment:
since it is a dependency of `can-vdom` and it doesn't look like it's used directly here.